playbooks/common/openshift-master/restart_services.yml

```diff
@@ -1,22 +1,4 @@
 ---
-- name: Restart master API
-  service:
-    name: "{{ openshift.common.service_type }}-master-api"
-    state: restarted
-  when: openshift_master_ha | bool
-- name: Wait for master API to come back online
-  wait_for:
-    host: "{{ openshift.common.hostname }}"
-    state: started
-    delay: 10
-    port: "{{ openshift.master.api_port }}"
-    timeout: 600
-  when: openshift_master_ha | bool
-- name: Restart master controllers
-  service:
-    name: "{{ openshift.common.service_type }}-master-controllers"
-    state: restarted
-  # Ignore errrors since it is possible that type != simple for
-  # pre-3.1.1 installations.
-  ignore_errors: true
-  when: openshift_master_ha | bool
+- include_role:
```

Comment:
This one task could be moved up to the two places where restart_services.yml is called and do away with this file.  It may be good to keep the filename in the role as restart_services.yml because it focuses on just a service restart.

Reply:
I thought about that.  I didn't want to step over your re-ogranization of entry points.